### PR TITLE
Delete tasks

### LINKED ISSUE #9: As a member, I want to remove a task assigned to me
So that I can focus on undone tasks

### PR DESCRIPTION
**Pull request by yash-chowdhary**
fixes #9 #10 
User can now delete tasks if he/she is the Assignor or Assignee of the task.
Testing done (will add more in next release)

### CODE REVIEW

File: src/main/java/seedu/club/model/ClubBook.java [hunk 1]
```diff
@@ -268,7 +279,15 @@ private void removeGroupFromMember(Group toRemove, Member member)
             + "See member#equals(Object).");
         }
     }
-    //@@author yash-chowdhary
+
+    /**
+     * Adds order to list of orders.
```
Comment (MuhdNurKamal) [inline src/main/java/seedu/club/model/ClubBook.java]: Adds task to list of tasks?

File: src/main/java/seedu/club/model/task/Description.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import static java.util.Objects.requireNonNull;
+import static seedu.club.commons.util.AppUtil.checkArgument;
+
+/**
+ * Refers to the Description of a Task
+ */
+public class Description {
+
+    public static final String DESCRIPTION_VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
+    public static final String MESSAGE_DESCRIPTION_CONSTRAINTS = "Description should a non-empty alphanumeric string";
```
Comment (MuhdNurKamal) [inline src/main/java/seedu/club/model/task/Description.java:14]: Can description be more than alphanumeric? Eg some ppl use @ for at and ! to show importance and any other non-alphanumeric characters

File: src/main/java/seedu/club/model/task/UniqueTaskList.java [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+        for (Task task : internalList) {
+            if (task.equals(toCheck)) {
+
+            }
+        }
+        return internalList.contains(toCheck);
+    }
+
+    /**
+     * Adds an Task to the list.
```
Comment (MuhdNurKamal) [inline src/main/java/seedu/club/model/task/UniqueTaskList.java]: Adds a Task

File: src/main/java/seedu/club/model/task/Status.java [hunk 4]
```diff
@@ -0,0 +1,36 @@
+package seedu.club.model.task;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Represents the status of a Task
+ */
+public class Status {
+
```
Comment (MuhdNurKamal) [inline src/main/java/seedu/club/model/task/Status.java:9]: Possibility to use enums instead in future releases to reduce complexity

Comment (yash-chowdhary) [reply src/main/java/seedu/club/model/task/Status.java:9]: Good idea! Will do that in future releases

File: src/main/java/seedu/club/model/task/UniqueTaskList.java (hunk 3 above)
Comment (yash-chowdhary) [reply src/main/java/seedu/club/model/task/UniqueTaskList.java]: 👍 thanks for pointing it out

File: src/main/java/seedu/club/model/ClubBook.java (hunk 1 above)
Comment (yash-chowdhary) [reply src/main/java/seedu/club/model/ClubBook.java]: Oops. Will change it 👍

File: src/main/java/seedu/club/model/task/Description.java (hunk 2 above)
Comment (yash-chowdhary) [reply src/main/java/seedu/club/model/task/Description.java:14]: Hmm, I guess I could do that. However, this is visible to everyone, so I thought it should be in a proper format

**Pull request merged**